<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InlineServerSetupWidget</class>
 <widget class="QWidget" name="InlineServerSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="input_name">
     <property name="toolTip">
      <string>Name used to identify the server</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This name is used to recognize the server and as an identifier for the server presets. If omitted it will become server:port&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_host">
     <property name="toolTip">
      <string>Server host or address</string>
     </property>
     <property name="whatsThis">
      <string>Host name or IP address of the Xonotic server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="input_port">
     <property name="toolTip">
      <string>Port Number</string>
     </property>
     <property name="whatsThis">
      <string>UPD Port number of the Xonotic server</string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>26000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_password">
     <property name="toolTip">
      <string>Rcon Password</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rcon password, it must match the cvar &lt;span style=&quot; font-weight:600;&quot;&gt;rcon_password&lt;/span&gt; as set on the Xonotic server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="input_secure">
     <property name="toolTip">
      <string>Rcon Secure Protocol</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Rcon secure protocol, must match the value of the cvar &lt;b&gt;rcon_secure&lt;/b&gt; in the Xonotic server.&lt;/p&gt;
&lt;ul&gt;
    &lt;li&gt;&quot;No&quot; (&lt;tt&gt;rcon_secure 0&lt;tt&gt;) the password is not encrypted and is the most reliable option.&lt;/li&gt;
    &lt;li&gt;&quot;Time Based&quot; (&lt;tt&gt;rcon_secure 1&lt;tt&gt;) hashes the command based on the system time, it requires to have the system clock to be synchronized with the server clock.&lt;/li&gt;
    &lt;li&gt;&quot;Challenge Based&quot; (&lt;tt&gt;rcon_secure 2&lt;tt&gt;) asks the server for challenges, secure but not always reliable.&lt;/li&gt;
&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
      <property name="icon">
       <iconset theme="security-low">
        <normaloff/>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time Based</string>
      </property>
      <property name="icon">
       <iconset theme="security-medium">
        <normaloff/>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Challenge Based</string>
      </property>
      <property name="icon">
       <iconset theme="security-high">
        <normaloff/>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_action">
     <property name="icon">
      <iconset>
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>input_host</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InlineServerSetupWidget</receiver>
   <slot>update_placeholder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_port</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InlineServerSetupWidget</receiver>
   <slot>update_placeholder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_placeholder()</slot>
 </slots>
</ui>
